src/app/components/BibPage/BibMainInfo.jsx

```diff
@@ -1,34 +1,75 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Link } from 'react-router';
 import { isEmpty as _isEmpty } from 'underscore';
 
 const BibMainInfo = ({ bib = {} }) => {
   if (_isEmpty(bib)) return null;
 
-  const title = bib.title && bib.title.length ? bib.title[0] : '';
   const materialType = bib && bib.materialType && bib.materialType[0] ?
     bib.materialType[0].prefLabel : null;
+  const author = bib.creatorLiteral && bib.creatorLiteral.length ?
+    `${bib.creatorLiteral[0]}` : '';
+  const contributor = bib.contributorLiteral && bib.contributorLiteral.length ?
+    bib.contributorLiteral : null;
```

Comment:
Yeah I'm also not sure about what the desired distinction is between these, from a product owner perspective -- I agree that it will likely change. I think the idea was "author is the first, contributor is all of them." I think Contributor is maybe getting reworded as Additional Authors, and then it will be author=0, this field would be=1...-1. If that makes sense.